runtime/renderer/xwalk_content_renderer_client.h

```diff
@@ -62,6 +62,14 @@ class XWalkContentRendererClient
                        GURL* new_url) override;
 
   void AddKeySystems(std::vector<media::KeySystemInfo>* key_systems) override;
+  bool HandleNavigation(content::RenderFrame* render_frame,
```

Comment:
This function should be wrapped with `#ifdef OS_ANDROID  #endif`, so build failed on Linux.